test/petab/test_petab_import.py

```diff
@@ -76,6 +76,13 @@ def test_3_optimize(self):
             self.assertTrue(np.isfinite(
                 result.optimize_result.get_for_key('fval')[0]))
 
+    def test_4_gradients(self):
```

Comment:
```suggestion
    def test_check_and_log_gradients(self):
```

Reply:
Ah I thought you want to enumerate the test in the class, as the previous tests are enumerated. Changed!

Reply:
Ah, didn't notice sorry, fine as before then.